srcpkgs/pipewire/files/pipewire/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+mkdir -p /run/pipewire
+chown pipewire:pipewire /run/pipewire
```

Comment:
maybe we could do this in a single command via `install`, (see dbus service). But then we also need to assign a fixed id to user pipewire.

Reply:
I'm pretty sure it should be `_pipewire`.

Reply:
`install -o` can take both UIDs and names, ditto `-g`.

Reply:
that's true, my bad

Reply:
> I'm pretty sure it should be `_pipewire`.

I am not sure, I think _pipewire would be used for a group which users do not have to interact with (users should never be a part of it). Names without the leading _ like wheel are often used for groups which users might want to be a part of. If we agree on this, then the pipewire group shouldn't have a leading _ in its name.

Reply:
I believe the namespacing is more about avoiding conflicts than specifying what a user should or shouldn't touch. Unless it's a common name (like http or plugdev), creating a user name without a leading underscore should always be harmless.